packages/ui-manager/src/translation/useDynamicTranslation.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect } from 'react';
+import getTranslations from './translations';
+import { useTranslation } from 'react-i18next';
+import i18next from 'i18next';
+import { Locale } from '@ovhcloud/msc-utils';
+
+export const useDynamicTranslation = (namespace: string) => {
+  const { t, i18n } = useTranslation(namespace);
+
+  useEffect(() => {
+    const loadTrad = async () => {
```

Comment:
if the translation file doesn't exist you should load the default fallback language translation file (so i18next can fallback to the default language)

Reply:
@frenautvh we already have in the app a default fallback language as 
```
          fallbackLng: 'fr_FR',
````

In `manager/packages/manager/apps/container/src/index.tsx`

Reply:
seen together, there shouldn't have missing translations for components so we can skip the handling of fallback languages for now